lib/perl/Genome/Sys/Lock.pm

```diff
@@ -151,30 +159,63 @@ sub with_default_lock_resource_args {
 
     $args{block_sleep} = 60 unless defined $args{block_sleep};
     $args{max_try} = 7200 unless defined $args{max_try};
+    $args{scope} = $ENV{GENOME_SYS_LOCK_DEFAULT_SCOPE} unless defined $args{scope};
```

Comment:
I think, for now, I would vote for removing the environment variable and just hard-coding a default.  If we need to override the default for tests we can override a `default_scope()` method.

Going the warning route would require that change being a separate PR so it can trickle down to LIMS before this PR get deployed.  If we went the warning route I think I would vote more for providing `genome config validate` or something that would allow people to verify config changes independent of running any `genome` command.